src/front/glsl/parser.rs

```diff
@@ -1071,6 +1072,27 @@ pomelo! {
             );
             if let Some(id) = id {
                 extra.lookup_global_variables.insert(id, h);
+            } else {
+                // variables in interface blocks without an instance name are in the global namespace
+                // https://www.khronos.org/opengl/wiki/Interface_Block_(GLSL)
+                if let TypeInner::Struct { members } = &extra.module.types[d.ty].inner {
```

Comment:
Let's not match on a reference, instead we should just do `ref members`. This is more explicit and less sugary.